examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3035,7 +3035,7 @@ class GLTFParser {
 
 			}
 
-			if ( encoding !== undefined ) {
+			if ( encoding !== undefined && texture ) {
```

Comment:
Would it be better to bail at the top of this function instead? I'm thinking that `extendTexture` may also fail in the `KHR_texture_transform` case.

Reply:
Makes sense!